rpi_sensors_exporter/config_loader.py

```diff
@@ -0,0 +1,89 @@
+import os
+import argparse
+import logging
+import sys
+
+import yaml
+from schema import Schema, And, Optional, SchemaError
+
+logger = logging.getLogger("sensors_exporter")
```

Comment:
I don't know if this is a good advice but
logger = logging.getLogger("sensors_exporter") 
can become 
logger = logging.getLogger( __name__ )
And that way if you call it from another app it will log correctly.